frontend/src/scenes/data-warehouse/externalDataSourcesLogic.ts

```diff
@@ -62,4 +62,7 @@ export const externalDataSourcesLogic = kea<externalDataSourcesLogicType>([
             }
         },
     })),
+    afterMount(({ actions }) => {
```

Comment:
If we're loading it specifically on datawarehousescene logic, I think this change won't be needed